changelog/argument-mismatch.dd

```diff
@@ -0,0 +1,22 @@
+Argument mismatch errors have been improved
+
+$(P `dmd` now shows which argument failed to match a parameter with an explanatory error message.)
```

Comment:
FYI: this `P` isn't necessary for changelog entries. The wrapper is smart enough to add them automatically for you to provide a "Markdownesque experience".